src/components/pages/SurveyPage.css

```diff
@@ -31,21 +27,16 @@ h5 {
   text-align: left;
 }
 
-.Section-Header {
-  text-align: left;
-  margin-left: 20em;
-  margin-right: 20em;
-}
 
 .actions {
   display: block;
   margin-left: 1em;
   margin-right: 1em;
 }
 
-.actions button {
```

Comment:
Are you using this? If not we can delete! (Unless there's a special future purpose for keeping it.) 😄